src/quantuminspire/api.py

```diff
@@ -362,46 +557,60 @@ def _create_asset(self, name, project, content):
 
     #  other  #
 
-    def _wait_for_completed_job(self, quantum_inspire_job, collect_max_tries, sec_retry_delay=0.5):
-        """ Holds the process and requests the job status untill completed or when
+    def _wait_for_completed_job(self, quantum_inspire_job: QuantumInspireJob, collect_max_tries: int,
+                                sec_retry_delay: float = 0.5) -> bool:
+        """ Holds the process and requests the job status until completed or when
             the maximum number of tries has been reached.
 
         Args:
-            quantum_inspire_job (QuantumInspireJob): A job object.
-            collect_max_tries (int): The maximum number of request tries.
-            sec_retry_delay (float): The time delay in between requests in seconds.
+            quantum_inspire_job: A job object.
+            collect_max_tries: The maximum number of request tries.
+            sec_retry_delay: The time delay in between job status checks in seconds.
 
         Returns:
-            Boolean: True if the job result could be collected else false.
+            True if the job result could be collected else False.
         """
         attempts = itertools.count() if collect_max_tries is None else range(collect_max_tries)
         for attempt in attempts:
             time.sleep(sec_retry_delay)
-            status_message = '(id {}, iteration {})'.format(quantum_inspire_job.get_job_identifier(), attempt)
+            status_message = '(id: {}, iteration: {})'.format(quantum_inspire_job.get_job_identifier(), attempt)
```

Comment:
It seems `status_message` is not used, can't we just remove it?

Reply:
Ok, i think it was there for debugging reasons, to get feedback while waiting for the job to complete.